js/views/modals/purchase/Moderators.js

```diff
@@ -60,7 +60,7 @@ export default class extends baseVw {
     const op = this.options;
     const includeString = op.include ? `&include=${op.include}` : '';
     const urlString =
-      `ob/${op.apiPath}?async=${!!op.async}${includeString}&usecache=${!!op.useCache}`;
```

Comment:
I forgot I did that. You're right, that's a better place to put it than checking in the url string.